### PR TITLE
add SC election to proposal monitor

### PR DESCRIPTION
**Pull request by DZGoldman**

### CODE REVIEW

**Comment by DZGoldman** [comment]
hrm the GovernorExecuteStage status checker needs to wait for the vetting period to be over before executing for the nominee gov..

File: package.json [hunk 1]
```diff
@@ -42,11 +42,11 @@
...
     "coverage:refresh": "yarn coverage:filtered-report && yarn coverage:htmlreport",
     "propmon:ui": "cd src-ts && http-server . -p 8080",
-    "propmon:service": "ts-node ./src-ts/proposalMonitorCli.ts --jsonOutputLocation ./src-ts/propMonUi/proposalState.json --l1RpcUrl $ETH_RPC --govChainRpcUrl https://arb1.arbitrum.io/rpc --novaRpcUrl https://nova.arbitrum.io/rpc --coreGovernorAddress 0xf07DeD9dC292157749B6Fd268E37DF6EA38395B9 --treasuryGovernorAddress 0x789fC99093B09aD01C34DC7251D0C89ce743e5a4 --sevenTwelveCouncil 0x895c9fc6bcf06e553b54A9fE11D948D67a9B76FA --pollingIntervalSeconds 1",
+    "propmon:service": "ts-node ./src-ts/proposalMonitorCli.ts --jsonOutputLocation ./src-ts/propMonUi/proposalState.json --l1RpcUrl https://mainnet.infura.io/v3/8838d00c028a46449be87e666387c71a --govChainRpcUrl https://arb1.arbitrum.io/rpc --novaRpcUrl https://nova.arbitrum.io/rpc --coreGovernorAddress 0xf07DeD9dC292157749B6Fd268E37DF6EA38395B9 --treasuryGovernorAddress 0x789fC99093B09aD01C34DC7251D0C89ce743e5a4 --sevenTwelveCouncil 0x895c9fc6bcf06e553b54A9fE11D948D67a9B76FA --nomineeElectionGovernorAddress 0x8a1cDA8dee421cD06023470608605934c16A05a0  --pollingIntervalSeconds 1",
```
Comment (yahgwai) [inline package.json:45]: Do you want to leave your infura address in here?

File: src-ts/proposalStage.ts [hunk 2]
```diff
@@ -131,6 +133,124 @@ export enum ProposalStageStatus {
   TERMINATED = 4,
 }
 
+export class GovernorExecuteStage implements ProposalStage {
```
Comment (yahgwai) [inline src-ts/proposalStage.ts:136]: Is this different from the governor queue stage? Cant we just use that?

Comment (DZGoldman) [reply src-ts/proposalStage.ts:136]: The nominee and member election have no timelock, so the execute method is different (and something needs to be done about status)

Comment (DZGoldman) [reply src-ts/proposalStage.ts:136]: thinking maybe having a single GovernorExecuteStage that determines the governor type (timelock, no timelock + vetting period, no timelock + no vetting period) and handles status/execute accordingly is cleanest?

Comment (DZGoldman) [reply src-ts/proposalStage.ts:136]: oh also getExecuteReceipt is different (looks for ProposalExecuted vs. CallScheduled).

Comment (DZGoldman) [reply src-ts/proposalStage.ts:136]: so idk

File: src-ts/securityCouncilElectionCreator.ts [hunk 3]
```diff
@@ -0,0 +1,54 @@
+import { Wallet, BigNumber } from "ethers";
+import { SecurityCouncilNomineeElectionGovernor__factory } from "../typechain-types";
+import { JsonRpcProvider } from "@ethersproject/providers";
+import { getL1BlockNumberFromL2 } from "./utils";
+
+export class SecurityCouncilElectionCreator {
+  retryTime = 10 * 1000;
```
Comment (yahgwai) [inline src-ts/securityCouncilElectionCreator.ts:7]: can be public readonly

**Review by yahgwai — APPROVED** [review]

**Pull request merged**